airbyte-webapp/src/components/base/Tooltip/Tooltip.tsx

```diff
@@ -60,24 +63,31 @@ export const Tooltip: React.FC<TooltipProps> = (props) => {
         ref={reference}
         className={styles.container}
         style={disabled ? undefined : { cursor }}
-        onMouseOver={onMouseOver}
-        onMouseOut={onMouseOut}
+        onFocus={onFocus}
+        onBlur={onBlur}
+        onMouseOver={onFocus}
+        onMouseOut={onBlur}
+        aria-details={tooltipId}
```

Comment:
I think we should use the `aria-describedby` here instead of `aria-details`. The main difference is that `aria-describedby` will automatically be turned into the "description" of this element, and therefore read out by assistive technology after a short break after the name (if not prevented). This would also follow the [ARIA authoring guidelines for tooltips](https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/).